appserver/admingui/payara-console-extras/src/main/resources/fish/payara/admingui/extras/Strings.properties

```diff
@@ -94,6 +94,8 @@ hazelcast.members.members =Data Grid Instances
 hazelcast.members.membersHelp=The Data Grid Server Instances
 hazelcast.configuration.dynamic=Restart Data Grid
 hazelcast.configuration.dynamicHelp=Applies the data grid configuration change immediately, without needing a restart of the Server.<br/>WARNING this can cause data grid wide data loss if a large number of instances restart the data grid immediately.
+hazelcast.configuration.enableAutoPortSelection=Port Auto Increment
+hazelcast.configuration.enableAutoPortSelectionHelp=When enabled the data grid automatically increments the port to next available port should the start port be occupied already.
```

Comment:
I'd make the following changes to this:
"When enabled the data grid automatically increments the port to \***the**\* next available \*~~port~~\* should the start port be occupied  \*~~already~~\*."

I think it reads a bit better like that.